src/main/scala/strawman/collection/BitSet.scala

```diff
@@ -12,8 +12,6 @@ trait BitSetOps[+C <: BitSet with BitSetOps[C]]
   extends SortedSetOps[Int, SortedSet, C] {
   import BitSetOps._
 
-  protected[this] def coll: C
```

Comment:
This refinement of the type of `coll` is now unnecessary because all `Iterable`s have a `c: C` member (see below).